src/chip.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import {createUseStyles} from './jss';
+import {useTheme} from './hooks';
+import Box from './box';
+import {Text2} from './text';
+import IconButton from './icon-button';
+import IconCloseRegular from './generated/mistica-icons/icon-close-regular';
+
+const useStyles = createUseStyles(({colors}) => ({
+    container: {
+        display: 'inline-flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        border: `1px solid ${colors.border}`,
+        borderRadius: 16,
+    },
+}));
+
+type ChipProps = {
+    value: string;
+    startIcon?: React.ReactNode;
+    onClose?: (value: string) => void;
```

Comment:
why is the value needed?

Reply:
I use this value to remove this chip from an array of already selected options, also I could read this value from the context of a map (I use this component with the well-known chips.map(value => ....), so, I could remove it from here if you want